controller/share_manager_controller.go

```diff
@@ -287,11 +287,7 @@ func (c *ShareManagerController) syncShareManager(key string) (err error) {
 		if !c.shouldClaimOwnership(sm) {
 			return nil
 		}
-
-		// on owner ship transfer we set the state to unknown,
-		// which leads to cleaning up any of the old pods
 		sm.Status.OwnerID = c.controllerID
-		sm.Status.State = types.ShareManagerStateUnknown
```

Comment:
Why do we need to remove this?

Reply:
We don't want to clean up the IM pod when transfer ownership to a new node. Doing so would disrupt the current running workloads.